{"name": "Вход с несуществующим юзером", "status": "passed", "description": "Проверяем попытку входа с отправкой несуществующих данных", "start": 1736846655680, "stop": 1736846655932, "uuid": "bafdf848-67c0-47d2-9d7e-ba84d85f2fb3", "historyId": "2a06b3c443fe2c81b22f6ddff3379aba", "testCaseId": "2a06b3c443fe2c81b22f6ddff3379aba", "fullName": "tests.test_courier.TestCourierLoginAPI#test_courier_login_no_such_user_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourierLoginAPI"}, {"name": "host", "value": "LAPTOP-B6TT34TV"}, {"name": "thread", "value": "17268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}